eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig

```diff
@@ -6,36 +6,38 @@
  #}
 
 {% block ezstring_settings %}
-<ul class="fielddef-settings {{ fielddefinition.fieldTypeIdentifier }}-settings">
+<ul class="ez-fielddefinition-settings {{ fielddefinition.fieldTypeIdentifier }}-settings">
```

Comment:
actually we should also apply the prefix on the second class, so this should be something like:
```jinja
<ul class="ez-fielddefinition-settings ez-fielddefintion-{{ fielddefinition.fieldTypeIdentifier }}-settings">
```